tests/test_codelists.py

```diff
@@ -53,7 +66,17 @@ def codelists_path(tmp_path):
     yield tmp_path
 
 
-def test_codelists_check(codelists_path):
+def test_codelists_check(mock_check, codelists_path):
+    mock_check(response={"status": "ok"})
+    os.chdir(codelists_path)
+    codelists.check()
```

Comment:
Why isn't there an `assert` here?

Reply:
Because I copied it from another test that didn't assert?  If there's a problem, `codelists.check()` will raise an error, so it doesn't need an _assert_ in order to fail the test. I  guess it could have one, just to be more explicit about it